build/configs/rtl8720e/hello/defconfig

```diff
@@ -967,7 +967,7 @@ CONFIG_MTD_SMART_WEAR_LEVEL=y
 CONFIG_MM_KERNEL_HEAP=y
```

Comment:
I think CONFIG_RAM_KREGIONx_START & CONFIG_RAM_KREGIONx_SIZE should be updated accordingly